compiler/qsc/benches/library.rs

```diff
@@ -6,7 +6,9 @@ use qsc::{compile, PackageStore};
 
 pub fn library(c: &mut Criterion) {
     let store = PackageStore::new(compile::core());
-    c.bench_function("Standard library", |b| b.iter(|| compile::std(&store)));
+    c.bench_function("Standard library", |b| {
+        b.iter(|| compile::std(&store, qsc::Target::Full))
```

Comment:
The fragment `qsc::Target::Full` in this line makes me feel that the same thing has 2 different fully qualified names:
* `qsc::Target`
* `qsc_frontend::compile::Target`

(if that's the case then you may want to unify - use only one fully qualified name)

Reply:
This is intentional, as there are a bunch of types that are re-exported from `qsc`. The expectation is that code outside of the compiler folder uses `qsc` while inside it uses `qsc_frontend::compile` to get to this struct. This was one exception, but I can clean it up to make it consistent with the others.